e2e/fixtures.ts

```diff
@@ -0,0 +1,20 @@
+import {Page} from '@playwright/test'
+
+export enum Editor {
+  schema = 0,
+  value = 1,
+}
```

Comment:
I tend to avoid enum in ts.  https://www.youtube.com/watch?v=jjMbPt_H3RQ

Reply:
Yeah I know but let's consider this more carefully because in the end we'll still have to have a mapper to match the parameter to a number

Reply:
removed since as you pointed out, the editors can be more than two